pkg/ast/spl/tests/splParser_test.go

```diff
@@ -10132,8 +10009,62 @@ func Test_FillNull_ValueArg_FieldList(t *testing.T) {
 	assert.Equal(t, []string{"field1", "field2"}, aggregator.OutputTransforms.LetColumns.FillNullRequest.FieldList)
 }
 
-func performCommon_aggEval_BoolExpr(t *testing.T, query []byte, measureFunc utils.AggregateFunctions, strEnc string) {
+func GetMeasureFuncStr(measureFunc utils.AggregateFunctions) (string, string) {
```

Comment:
nit: since this is just for testing, it can be private
```suggestion
func getMeasureFuncStr(measureFunc utils.AggregateFunctions) (string, string) {
```

Reply:
addressed in the latest commit.